### PR TITLE
Create ru_RU.lang for Design

### PR DESCRIPTION
**Pull request by SawFowl**
Binniedesign

### CODE REVIEW

**Comment by KorDum** [comment]
I will oversee this translation.

File: design/src/main/resources/assets/binniedesign/lang/ru_RU.lang [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+binniedesign.pattern.ornate.striped.corner=Изысканный полосатый угол
+binniedesign.pattern.testing.block=Тестовый блок
+binniedesign.pattern.diagonal.halved=Диагональ пополам
+binniedesign.pattern.cornered.diagonal=Угловая диагональ
+binniedesign.pattern.opposite.cornered.diagonal=Противоположная угловая диагональ
+binniedesign.pattern.thick.cornered.diagonal=Толстая угловая диагональ
+binniedesign.pattern.thin.barred.end=Тонкий разделенный конец
+binniedesign.pattern.thick.barred.end=Толстый разделенный конец
+binniedesign.pattern.overlapped.barred=Перекрытое разделение
+binniedesign.pattern.overlapped.split.barred=Перекрытое разделение скрещивания
```
Comment (KorDum) [inline design/src/main/resources/assets/binniedesign/lang/ru_RU.lang]: Перекрытое разделение со скрещиванием

**Review by KorDum — CHANGES_REQUESTED** [review]

**Review by KorDum — APPROVED** [review]

**Comment by KorDum** [comment]
@Mr-Krab thanks for hardwork!
@mezz all done for merge :)

**Comment by mezz** [comment]
Thank you!

**Pull request merged**